snuba_sdk/mql/mql.py

```diff
@@ -546,3 +593,9 @@ def visit_identifier(self, node: Node, children: Sequence[Any]) -> str:
     def generic_visit(self, node: Node, children: Sequence[Any]) -> Any:
         """The generic visit method."""
         return children
+
+
+@dataclass
+class FilterFactorValue(object):
```

Comment:
nit: no need to call it `*Value`, since it contains both value and metadata.
```suggestion
class FilterFactor(object):
```

Reply:
Changed